CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 This changelog will be used to generate documentation on [release notes page](http://azure.microsoft.com/en-us/documentation/articles/app-insights-release-notes-dotnet/).
 
+## Version 2.0.0-beta4
+- RoleInstance and NodeName by the default are initialized with the machine name.
+
```

Comment:
Why is the ikey from environment variable change in the beta3 changelog when the changes appear in this PR?